lib/ansible/modules/storage/netapp/netapp_e_lun_mapping.py

```diff
@@ -37,6 +39,18 @@
     description:
       - The name of the volume you wish to include in the mapping.
     required: True
+    aliases:
+      - volume
+  lun:
+    description:
+      - This option is deprecated and will have no effect on the module's outcome
```

Comment:
Confused, why has this been added if it's deprecated?

Reply:
The previous merge removed the two options, lun and target_type, that should have been deprecate instead for backwards capability.  How should I correct my mistake?

Reply:
So if these options don't do anything that seems to break backwards compatability

Reply:
If there are existing customer plays that contain the two options, the module will still work as expected but we simplified the logic such that the two options are not needed.  The netapp e-series restapi already had the capability to determine those options without customers explicitly stating them.

Reply:
Could the existing customer set these deprecated options to something other than what the restapi would automatically do, ie surprise the user? 

If these options don't do anything then removing them maybe the right decision. A `note:` should be added in the he modules docs and in `porting_guide_2.7.rst`